stats/stats/src/charts/lines/mod.rs

```diff
@@ -88,6 +88,7 @@ pub use new_native_coin_transfers::{
     NewNativeCoinTransfers, NewNativeCoinTransfersInt, NewNativeCoinTransfersMonthly,
     NewNativeCoinTransfersWeekly, NewNativeCoinTransfersYearly,
 };
+pub(crate) use new_txns::NewTxnsStatement;
```

Comment:
either replace all reimports or change to `pub use` for consistency

Reply:
why? it's only needed within the crate. it is not meant to be exposed for `server` part, for example

Reply:
resolving bc vacation + answer given 🙈